docs/Training-ML-Agents.md

```diff
@@ -134,12 +134,11 @@ environment, you can set the following command line options when invoking
   [Academy Properties](Learning-Environment-Design-Academy.md#academy-properties).
 * `--train` – Specifies whether to train model or only run in inference mode.
   When training, **always** use the `--train` option.
-* `--num-envs` - Specifies the number of parallel environments to collect
+* `--num-envs=<n>` - Specifies the number of concurrent, parallel Unity environment instances to collect
   experiences from when training. Defaults to 1.
-* `--base-port` - Specifies the starting port for environment workers. Each Unity
-  environment will use the port `(base_port + worker_id)`, where the worker ID
-  are sequential IDs given to each environment from 0 to `num_envs - 1`.
-  Defaults to 5005.
+* `--base-port` - Specifies the starting port for each concurrent, parallel Unity environment instance. Each instance will use the port `(base_port + worker_id)`, where the `worker_id`
```

Comment:
This is a bit confusing: "the starting port for each concurrent, parallel Unity environment instance".  This is the starting port from which all other ports are given sequential port numbers.  Not the starting port for each.